control/server.py

```diff
@@ -126,7 +140,7 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_value, traceback):
         """Cleans up SPDK and server instances."""
         if exc_type is not None:
-            self.logger.exception("GatewayServer exception occurred:")
+            self.logger.exception(f"GatewayServer exception occurred:\n{traceback}\n")
```

Comment:
`log.exception()` already attaches the traceback (newline-separated):
```suggestion
            self.logger.exception(f"GatewayServer exception occurred")
```

Reply:
@epuertat this didn't work for some reason so I had to do it explicitly. I guess the stack structure here is different so the logger.exception() function failed to identify the stack trace.